### PR TITLE
update variant-detail-card on user entered phenotype

### PR DESCRIPTION
**Pull request by WilliamRichards2017**
When a user enters a phenotype and replaces the gene list, the variant-detail-card is updated to display the newly selected variant (or to not display if there are no variants in the replaced gene list)

### CODE REVIEW

**Comment by anderspitman** [comment]
quarantest build successful. Demo link at
http://2a784464.quarantest.iobio.io

**Comment by WilliamRichards2017** [comment]
To Test:
1) Click on demo data
2) Select RAI1 pathogenic variant
3) Type 'charcot marie tooth disease' in phenotype search
4) Click 'replace gene list' in popup
5) The variant detail card should no longer display the variant from RAI1

**Comment by mvelinder** [comment]
> quarantest build successful. Demo link at
> http://2a784464.quarantest.iobio.io

Not working for me

File: client/app/components/pages/GeneHome.vue [hunk 1]
```diff
@@ -3893,6 +3905,7 @@ export default {
 
             self.toClickVariant = firstFlaggedVariant;
             self.showLeftPanelWhenFlaggedVariants();
+            console.log("selecting first flagged variant");
```
Comment (tonydisera) [inline client/app/components/pages/GeneHome.vue:3908]: Please remove debug statements.

File: client/app/components/pages/GeneHome.vue [hunk 2]
```diff
@@ -1115,6 +1115,11 @@ export default {
   },
 
   watch: {
+
+    selectedVariant: function(){
+      console.log("selected variant in watcher", this.selectedVariant);
```
Comment (tonydisera) [inline client/app/components/pages/GeneHome.vue:1120]: Please do a diff to see that debug messages, empty lines, etc aren't committed.

**Comment by anderspitman** [comment]
quarantest build successful. Demo link at
http://26241323.quarantest.iobio.io

**Comment by anderspitman** [comment]
quarantest build successful. Demo link at
http://c128e747.quarantest.iobio.io

**Pull request merged**